box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/InviteCollaboratorsFragment.java

```diff
@@ -111,8 +122,12 @@ public boolean areCollaboratorsPresent() {
         return false;
     }
 
-    public void setInviteCollaboratorsListener(InviteCollaboratorsListener listener) {
-        mInviteCollaboratorsListener = listener;
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        // Attach the listener to view once createView is complete
+        mInviteCollaboratorsListener = ((InviteCollaboratorsListener) getActivity());
```

Comment:
I don't like much this assumption that the activity using this Fragment implements the InviteCollaboratorsListener, I know it is true here. But in case someone wants to reuse this fragment somewhere else, then this dependency is not clean and will only fail on runtime.